src/index.ts

```diff
@@ -140,7 +143,12 @@ function getHandlers(
           ? props.delta
           : props.delta[dir.toLowerCase() as Lowercase<SwipeDirections>] ||
             defaultProps.delta;
-      if (absX < delta && absY < delta && !state.swiping) return state;
+      if (absX < delta && absY < delta && !state.swiping)
+        return { ...state, initialDirection: state.initialDirection || dir };
```

Comment:
🤔 i think we'd need to actually set `initialDirection` outside of the catch for `delta`. We should probably only count the initial direction once the user has surpassed the delta which signifies the start of a "swipe" or "swiping"